addons/website_sale_slides/models/product_product.py

```diff
@@ -16,4 +16,4 @@ def get_product_multiline_description_sale(self):
             return super(Product, self).get_product_multiline_description_sale()
 
         new_line = '' if len(payment_channels) == 1 else '\n'
-        return _('Access to:%s%s', new_line, '\n'.join(payment_channels.mapped('name')))
+        return _('Access to: %s%s', new_line, '\n'.join(payment_channels.mapped('name')))
```

Comment:
LGTM, however, I am wondering if we can be sure the name_get of SOL will stay in one line or not in the case `payment_channels` is greater than 1? 🤔

Reply:
> LGTM, however, I am wondering if we can be sure the name_get of SOL will stay in one line or not in the case `payment_channels` is greater than 1? thinking

I have added a log note to the task with an attachment.Please look at.thanks